L1Trigger/L1TTwinMux/src/IOPrinter.cc

```diff
@@ -127,10 +126,10 @@ void IOPrinter::run(
         for (station=1; station<=4; station++){
           int nhits = DTRPCBxCorrection::noRPCHits(*inphiDigis, bx, wheel, sector, station);
           for(int hit=0; hit<nhits; hit++){
-          dtts1=0;
-          dtts1 = inphiDigis_tm->chPhiSegm(wheel,station,sector,bx,hit);
-          if(dtts1) {
-            cout<<dtts1->bxNum()<<"\t"<<dtts1->whNum()<<"\t"<<dtts1->scNum()<<"\t"<<dtts1->stNum()<<"\t"<<dtts1->phi()<<"\t"<<dtts1->phiB()<<"\t"<<dtts1->code()<<"\t"<<dtts1->Ts2Tag()<<"\t"<<dtts1->BxCnt()<<"\t0"<<endl;
+            dtts1=0;
```

Comment:
I think this is not needed. Can `inphiDigis_tm->chPhiSegm(wheel,station,sector,bx,hit)` give back a null pointer?

Reply:
Yes I think it is possible. It is used also in other instances inside the code